docs/login_and_acl.md

```diff
@@ -134,6 +135,21 @@ Ensure you run `azd env set AZURE_USE_AUTHENTICATION` to enable the login UI onc
 
 In both the chat and ask a question modes, under **Developer settings** optional **Use oid security filter** and **Use groups security filter** checkboxes will appear. The oid (User ID) filter maps to the `oids` field in the search index and the groups (Group ID) filter maps to the `groups` field in the search index. If `AZURE_ENFORCE_ACCESS_CONTROL` has been set, then both the **Use oid security filter** and **Use groups security filter** options are always enabled and cannot be disabled.
 
+#### Programmatic Access with Authentication
+
+If you want to use the chat endpoint without the UI and still use authentication, you must disable the integrated app services authentication and use the authentication built into the sample app instead. Ensure the `AZURE_DISABLE_APP_SERVICES_AUTHENTICATION` environment variable is set before deploying.
+
+Get an access token that can be used for calling the chat API using the following code:
+
+```python
+from azure.identity import DefaultAzureCredential
+import os
+
+token = DefaultAzureCredential().get_token(f"api://{os.environ['AZURE_SERVER_APP_ID']}/access_as_user", tenant_id=os.getenv('AZURE_AUTH_TENANT_ID', os.getenv('AZURE_TENANT_ID')))
```

Comment:
This code only works in prod, since AZURE_SERVER_APP_ID isnt exported locally. Is that intentional? If so, you should specify that this needs to be run in the App Service SSH.

Reply:
Nevermind! I forgot that we set that ourself, in the scripts.